src/input.js

```diff
@@ -105,6 +107,7 @@ class Input extends Component {
         aria-expanded={isOpen}
         autoComplete="off"
         value={(inputValue === null ? selectedItemValue : inputValue) || ''}
+        autoFocus={this.menu}
```

Comment:
It's just checking for the presence of `Menu` and then evaluates `autoFocus` to true. So no matter if `Button` opens it or not, it will autoFocus based on being a child of `Menu`.